src/components/ConfigViewer/columns.tsx

```diff
@@ -0,0 +1,63 @@
+import { CellContext, ColumnDef } from "@tanstack/table-core";
+import * as timeago from "timeago.js";
+
+export const defaultTableColumns: ColumnDef<any>[] = [
+  {
+    header: "Type",
+    accessorKey: "config_type"
+    // cellClass: `px-5 py-2`
+  },
+  {
+    header: "Name",
+    accessorKey: "name"
+    // cellClass: `px-5 py-2`
+  },
+  {
+    header: "Tags",
+    accessorKey: "tags",
+    cell: TagsCell
+    // cellClass: `px-5 py-2`
+  },
+  {
+    header: "Created",
+    accessorKey: "created_at",
+    cell: DateCell
+    // cellClass: `px-5 py-2`
+  },
+  {
+    header: "Last Updated",
+    accessorKey: "updated_at",
+    cell: DateCell
+    // cellClass: `px-5 py-2`
+  }
+];
+
+export function TagsCell({ row, column }: CellContext<any, any>): JSX.Element {
+  const tags = row?.getValue<any[]>(column.id);
+
+  return (
+    <div className="flex">
+      {tags?.length > 0 ? (
+        tags?.map((tag) => (
+          <div
+            className="bg-gray-200 px-1 py-0.5 mr-1 rounded-md text-gray-600 font-semibold text-xs"
+            key={tag}
+          >
+            {tag}
+          </div>
+        ))
+      ) : (
+        <span className="text-gray-400">none</span>
+      )}
+    </div>
+  );
+}
+
+export function DateCell({ row, column }: CellContext<any, any>) {
+  const dateString = row?.getValue<string>(column.id);
```

Comment:
We should be using utils for this

Reply:
Do you mean for the `getValue` method?

Reply:
please use `relativeDateTime` util method to convert date.